.github/workflows/pr-preview.yaml

```
@@ -0,0 +1,447 @@
+                --image-ids imageTag=pr-${pr_number} || true
+              
+              echo "Cleaned up PR #$pr_number deployment"
+            fi
+          done
+
+  cleanup-closed-pr:
+    if: github.event.action == 'closed'
+    runs-on: blacksmith-4vcpu-ubuntu-2204
+    steps:
+      - name: Cleanup PR deployment
+                run: |
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 70: mapping values are not allowed in this context

(syntax-check)